ivy.el

```diff
@@ -914,6 +914,20 @@ When this directory doesn't exist, return nil."
   "When non-nil, `ivy-partial-or-done' should insert a space."
   :type 'boolean)
 
+(defun ivy-complete-or-done()
+  "Complete the minibuffer text with selected.
+if selected text is directory, expand directory,
+and if the text has been completed, forward to `ivy-alt-done' when second call."
```

Comment:
Please start proper sentences with the first word capitalised. Please also [fill](https://www.gnu.org/software/emacs/manual/html_node/emacs/Filling.html) docstrings to the conventional `emacs-lisp-docstring-fill-column` via <kbd>M-q</kbd> (`fill-paragraph`).